tests/baselines/reference/parameterInitializersForwardReferencing1.errors.txt

```diff
@@ -57,4 +58,12 @@ tests/cases/conformance/functions/parameterInitializersForwardReferencing1.ts(29
     }
     
     function f8(foo1: string, bar = foo1) { }
+    
+    let bazzzz = 2;
+    function f9(bar = () => bazzzz) {
+                            ~~~~~~
+!!! error TS2373: Initializer of parameter 'bar' cannot reference identifier 'bazzzz' declared after it.

```

Comment:
why is this error here? It doesn't reference the inner `var bazzzz` (`var` doesn't work any different than `let` and `const` in this case). And the outer `let bazzzz` is definitely not declared after the parameter.